notebooks/en/annotate_text_data_transformers_via_active_learning.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>The title is too long, let's make it shorter (also in the toctree yaml file). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>"Annotate text data for transformers using active learning with Cleanlab" - what do you think about this? </p>

Reply:
I think you can even go with "Annotate text data using Active Learning with Cleanlab"

Reply:
<p>ok!</p>